TEAM_STRUCTURES.md

```diff
@@ -137,21 +143,8 @@ Tools and systems for the IPFS community.
 - Manage and maintain the IPFS pinning service, pinbot.
 - Monitor services and hosts used by the IPFS dev teams.
 - Provide wisdom to other users that want to host IPFS nodes.
-
-### Dev Team Enablement (QA, Testing, Support)
-
-- **Coordination**: https://github.com/ipfs/testing
-- **Captain**: [Victor Bjelkholm](https://github.com/VictorBjelkholm)
-
-The QA, Testing and Dev Team Enablement Working Group focuses on building developer tools, CI, and automated maintenance tasks to improve the developer experience of the many IPFS (and related) projects.
-
-**Responsibilities include**:
-- Enable developers to develop faster by reducing friction from dev.
-- Upgrade and build dev tools (i.e AEgir and gx).
 - Create a sound testing infrastructure for all the IPFS projects.
```

Comment:
This sounds like "we will run testing infrastructure" when I think our responsibility is more around "Enable dev teams to run reliable, efficient tests and manage continuous deployment```